lib/scrabble.rb

```diff
@@ -15,4 +13,19 @@ def point_values
       "Y"=>4, "Z"=>10
     }
   end
+
+
+  def score(word)
+    list_of_values = []
+    if word == '' || word == nil
+      list_of_values << 0
+    else
+      list_of_letters = word.upcase.chars
+      list_of_letters.each do |x|
+        list_of_values << @values[x]
+      end
+    end
+    list_of_values.reduce(:+)
```

Comment:
Like that you're erring on the side of longer variable names, but in this case would likely do away with the `list_of_` in both cases and just go with `letters` and `values`.